src/site/content/en/blog/model-viewer-10/index.md

```diff
@@ -1,18 +1,94 @@
 ---
+layout: post
 title: Announcing <model-viewer> 1.0
 subhead: 3D models for your web page as easily as writing HTML.
 authors:
   - joemedley
+  - rkochman
 description: 3D models are more relevant than ever. Retailers bring in-store shopping experiences to customers' home. Museums let anyone see artifacts from anywhere. 3D is difficult to do without a deep knowledge of 3D technologies or third-party hosting. <model-viewer> 1.0 makes these tasks as easy as writing HTML.
-date: 2020-06-26
+date: 2020-07-26
 tags:
   - blog
   - 3d
   - model-viewer
-hero: hero.jpg
-alt: A 3D model of a space suit.
 ---
 
-TBD
+3D models are more relevant than ever. Retailers are using 3D models to bring the in-store shopping experience to customers at home. Museums are making 3D models of their artifacts available to everyone on the web. Unfortunately, it can be difficult to add a 3D model to a website in a way that provides a great user experience, without a deep knowledge of 3D technologies or resorting to hosting 3D content on a third-party site. `<model-viewer>`, [introduced in early 2019](https://developers.google.com/web/updates/2019/02/model-viewer), seeks to make putting 3D models on the web as easy as writing a few lines of HTML. Since then, the team has been working to address feedback and requests from the community. The culmination of that work is `<model-viewer>` version 1.0. 
 
+## What's new since last year?
 
+Version 1.0 includes built-in support for augmented reality (AR) on the web, improvements to speed and fidelity, and other frequently-requested features.
+
+### Augmented reality
+
+Viewing a 3D model on a blank canvas is great, but being able to view it in your space is even better. Now `<model-viewer>` supports WebXR augmented reality, for an entirely-within-the-browser 3D and AR experience on Chrome Android. 
+
+```html
+<model-viewer src="ToyTrain.glb" ar ar-scale="auto" camera-controls alt="A 3D model of a wooden toy train"> 
+</model-viewer>
+```
+<iframe style="width:100%; height: 100%;position: absolute; top: 50%; left: 50%; transform: translate(-50%,-50%);" src="https://modelviewer.dev/webdotdev/webxr.html" frameborder="0" allowfullscreen></iframe>
```

Comment:
(Noting for myself) I wasn't sure if @robdodson would be OK with iframe-ing stuff into our content but from the looks of this content it seems OK: https://github.com/GoogleChrome/web.dev/pull/3289/files#r445240150

Reply:
Yeah if folks need to demo something then iframes are the way to go. That's how we pull our glitch examples into the site.

You'll want to change the inline styles on all of these iframes as they attempt to make it fill the entire viewport. I'd suggest something like this:

```html
  <iframe
    src="https://modelviewer.dev/webdotdev/webxr.html"
    title="A toy train using model-viewer"
    style="height: 420px; width: 100%; border: 0;">
  </iframe>
```